OWNERS

```diff
@@ -5,3 +5,9 @@ approvers:
   - dims
   - spiffxp
   - thockin
+  - hh
+
```

Comment:
@BobyMCbobs Please, maintain the list of the approvers only to SIG Chairs and TLs.

Reply:
OK, I've updated it